daemon/export_api_users_test.go

```diff
@@ -33,34 +33,23 @@ func MockHasUserAdmin(mockHasUserAdmin bool) (restore func()) {
 	}
 }
 
-func MockUserLookup(lookup func(username string) (*user.User, error)) (restore func()) {
-	oldLookup := userLookup
-	userLookup = lookup
+func MockDeviceStateCreateUser(createUser func(st *state.State, mgr *devicestate.DeviceManager, sudoer bool, createKnown bool, email string) (createdUsers []devicestate.UserResponse, internal_err bool, err error)) (restore func()) {
+	oldCreateUser := deviceStateCreateUser
```

Comment:
you can use testutil.Backup(&deviceStateCreateUser) here

Reply:
Yes, it's because it was added recently, and we didn't change all old code to use it. Thanks for updating your branch!